lib/surface/compiler.ex

```diff
@@ -47,6 +47,8 @@ defmodule Surface.Compiler do
     Surface.Directive.For
   ]
 
+  @directive_prefixes [":", "s-"]
```

Comment:
is this something we'd want to pull out into the config?

Reply:
Maybe in the future. Let's start with this and see if anyone will ever use it :)